src/RequestHeaderParser.php

```diff
@@ -29,7 +29,7 @@ public function feed($data)
             try {
                 $this->parseAndEmitRequest();
             } catch (Exception $exception) {
-                $this->emit('error', [$exception]);
+                $this->emit('error', [$exception, $this]);
```

Comment:
Tbh I have no problem reverting this line and associated tests. This line hasn't caused any issues. I'll revert it :+1: